sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -639,10 +740,15 @@ def test_preauthorize_user_with_user_delegation_key(self, datalake_storage_accou
                                              credential=sas_token)
 
         acl = new_file_client.set_access_control(permissions='0777')
-        self.assertIsNotNone(acl)
+        assert acl is not None
 
+    # TODO: MAYBE LIVE ONLY BC CONCURRENCY 2
```

Comment:
Probably remove the TODO :)

But to answer the question, the data size here is only 1024 which would be smaller than the default chunk size (and I don't think it's changed here) so it will download in a single chunk which means `max_concurrency` is doing nothing here. I think you can remove the `max_concurrency` and leave this recorded.

Applies to test below as well. I wouldn't be surprised if this occurs more in the file too.